### DISCUSSION TITLE
how to create Inline keyboard in aiogram?

### DISCUSSION THREAD

**Discussion by PrOkSSy** [community/community#44001] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Show & Tell

### Body

I need to add inline keyboard (yes or no). Plz help me!!!

**Comment by kalinin-egor** [comment, marked as answer]
Hello! 
this is not too hard:

from aiogram.types import InlineKeyboardMarkup, InlineKeyboardButton

profile_kb = InlineKeyboardMarkup(row_width=1)
profile_1 = InlineKeyboardButton(callback_data='yes', text='Yes')
profile_2 = InlineKeyboardButton(callback_data='no', text='No')
profile_kb.add(profile_1, profile_2)

**Reply by PrOkSSy** [reply]
thank you so much !!!

**Comment by cehfisher** [comment]
Hi @PrOkSSy - this issue appears to be resolved [per this comment](https://github.com/orgs/community/discussions/44001#discussioncomment-4660322). Could you please close this issue? Thank you! (cc @queenofcorgis)
